lib/Core/Tests/Input/JsonInputTest.php

```diff
@@ -66,6 +66,9 @@ public function provideSingleValuePairTests()
                                 'excluded-simple-values' => ['value3'],
                             ],
                         ],
+                        'order' => [
+                            '@date' => 'ASC',
```

Comment:
Do we need to enforce the fields are always prefixed with `@` here, do we do it ourselves in the JsonInput?

Reply:
I think we can handle this automatically, no need to expose internal details here.